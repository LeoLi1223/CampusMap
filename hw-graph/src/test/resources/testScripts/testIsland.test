# Tests that a graph contains isolated nodes, all nodes
# can be printed and all edges from each node can be printed

# Create a graph
CreateGraph island

# Add nodes
AddNode island n1
AddNode island n2
AddNode island n3

# Add Edges (isolate n3)
AddEdge island n1 n2 e12
AddEdge island n2 n1 e21

# print all nodes in the graph and all edges from each node
ListNodes island
ListChildren island n1
ListChildren island n2
ListChildren island n3