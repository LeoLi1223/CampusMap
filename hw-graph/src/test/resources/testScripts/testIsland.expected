# Tests that a graph contains isolated nodes, all nodes
# can be printed and all edges from each node can be printed

# Create a graph
created graph island

# Add nodes
added node n1 to island
added node n2 to island
added node n3 to island

# Add Edges (isolate n3)
added edge e12 from n1 to n2 in island
added edge e21 from n2 to n1 in island

# print all nodes in the graph and all edges from each node
island contains: n1 n2
the children of n1 in island are: n2(e12)
the children of n2 in island are: n1(e21)
the children of n3 in island are: